engine_comparison/README.md

```diff
@@ -2,20 +2,48 @@
 
 This is a set of scripts to run A/B testing among different fuzzing engines.
 
-## gcloud
+## Gcloud
+
+Currently, these scripts only run on Google Cloud, but support for alternatives will be incorporated.
+
+### Gcloud Usage
+
+Before running experiments, use `gcloud_creator.sh` to create the initial VM.
+This can take 45-60 seconds, but `gcloud_creator.sh` doesn't exit until google
+cloud is ready for `begin_experiment.sh` to run; one needs only to wait for the
+"Instance Created" output message.
+
+In general, `gcloud` commands don't complete until the task is fully finished,
+but there is sometimes a small latency.
+
+VM status can also be found on [the gcloud
+console](https://pantheon.corp.google.com/compute/instances?project=fuzzer-test-suite)
```

Comment:
Good point. As of now, several of our unique/private assets are programmed into these scripts, including the gcloud VM image which is stored privately in the cloud.